PrivateHeaders/AccessibilityUtilities/AXSettings.h

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
Id' rather remove the copyright header from this file, as this is an Apple system headers